### ISSUE TITLE
Missing tooltips in the new send media ui

### ISSUE THREAD

**Issue by MarcAnt01** [UnigramDev/Unigram#1529]

**Comment by FrayxRulez** [comment]
WHICH ONES

**Comment by MarcAnt01** [comment]
Done and go back, self-assigned tho
